src/firefly/js/visualize/ChartsTableViewPanel.jsx

```diff
@@ -438,58 +510,61 @@ class ChartsPanel extends React.Component {
 
     renderOptions() {
         const {optionsShown, chartType, heightPx} = this.state;
-        const { tblId, tableModel, tblStatsData} = this.props;
+        const { tblId, tableModel, tblStatsData, tblPlotData, tblHistogramData, optionsPopup, chartId} = this.props;
         if (optionsShown) {
-            return (
-                <div style={{display:'inline-block',overflow:'auto',width:(OPTIONS_WIDTH-20),height:heightPx,paddingLeft:10,verticalAlign:'top'}}>
-                    {this.renderChartSelection()}
-                    <OptionsWrapper  {...{tblId, tableModel, tblStatsData, chartType}}/>
-                </div>
-            );
-        } else {
-            return undefined;
+            if (!optionsPopup) {
```

Comment:
nitpicking.. this if condition can be easily combined with previous if statement.